src/styles/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
```

Comment:
same comment as somewhere here, consider using variables for the colors (and sizes/measurements too), so it can be reused and changed with ease.